CONTRIBUTING.md

```diff
@@ -55,11 +55,19 @@ the `generate-coverage.sh` script can be run from the root of this
 repository to avoid contaminating your environment, but note it will
 download a full toolchain on each run.
 
-## Website and web playground
+## Web site and web playground
 
 If you have [Deno](https://deno.land/) installed, you can start a local web
 server like this:
```

Comment:
I wonder if it's worth bundling something with the Nix stuff to include additional tools, like Deno and Graphviz? I know you can put them behind arguments in a normal `shell.nix`, but I don't know how it works with flakes.

Anyway, not for this PR regardless, but perhaps worth an issue...